apps/epicdev-ai/src/app/(commerce)/products/[slug]/page.tsx

```
@@ -0,0 +1,211 @@
+
+	if (user && purchaseForProduct) {
+		const { purchase, existingPurchase } =
+			await courseBuilderAdapter.getPurchaseDetails(
+				purchaseForProduct.id,
+				user.id,
+			)
+
+		productProps = {
+			...baseProps,
+			hasPurchasedCurrentProduct: Boolean(purchase),
+			...(Boolean(true)
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)